libs/components/layout/src/lib/modules/definition-list/definition-list-label.component.ts

```diff
@@ -12,7 +12,7 @@ import { SkyDefinitionListService } from './definition-list.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SkyDefinitionListLabelComponent {
-  public labelWidth: number;
+  public labelWidth: number | undefined;
```

Comment:
I'm seeing two inputs in `definition-list.component.ts` that need to accept `undefined`.

Reply:
fixed